src/resources/schema/cell-codeoutput.yml

```diff
@@ -91,6 +91,7 @@
     short: "Include line numbers in code block output (`true` or `false`)"
     long: |
       Include line numbers in code block output (`true` or `false`).
+      Limitation: does not work properly with `code-link` set to `true`.
```

Comment:
`code-line-numbers` always works. It is `code-link` which does not work if `code-line-numbers` is set. 

So I don't think adding that here in `code-line-numbers` reference page does really help. Also because it will be shown in other reference page than `html` and the limitation is only in HTML (as `code-link` is only for HTML and R documents)

Reply:
```suggestion
```

Reply:
Fair point, I overlooked and missed `formats: [$html-all, ms, $pdf-all]`.

Reverted in 3c185cda684c637464951da9f9bf106ff33473f5

Reply:
3c185cda684c637464951da9f9bf106ff33473f5

(VSCode PR extension did a weird thing with my replies...)